pkg/controller/operators/adoption_controller.go

```diff
@@ -0,0 +1,438 @@
+/*
+Copyright 2020 The Kubernetes authors.
```

Comment:
😲  this is not the right copyright - we also removed the year from all of our generated files (and don't typically have the notice on non-generated files).

We discussed using the short names everywhere but haven't done it yet.

Reply:
will remove -- this was me using kubebuilder defaults, sorry